proposals/gc/MVP.md

```diff
@@ -432,6 +432,7 @@ This extends the [encodings](https://github.com/WebAssembly/function-references/
 | -0x15  | `(ref ht)`      | `ht : heaptype (s33)` | from funcref proposal |
 | -0x16  | `i31ref`        |            | |
 | -0x17  | `(rtt n ht)`    | `n : u32`, `ht : heaptype (s33)` | |
+| -0x18  | `(rtt ht)`      | `ht : heaptype (s33)` | |
```

Comment:
Here and in the line above: s/heaptype (s33)/typeidx (u32)/

Reply:
Oops, fixed.